app/models/comment.rb

```diff
@@ -0,0 +1,14 @@
+class Comment < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] Again here we first need to establish the association between the User and the Comment correctly. We need to change the `belongs_to :user` line to 
  
 `belongs_to :author, class_name: 'User'`